### PR TITLE
Add JSCS, wikimedia preset with a few tweaks

### PR DESCRIPTION
**Pull request by ricordisamoa**

### CODE REVIEW

**Comment by ricordisamoa** [comment]
npm tests should not be mixed with the PHP matrix :disappointed:

File: build/travis/script.sh [hunk 1]
```diff
@@ -7,3 +7,5 @@ php phpunit.php -c ../../extensions/PropertySuggester/phpunit.xml.dist
 
 # cd ../wiki/extensions/PropertySuggester
```
Comment (filbertkm) [inline build/travis/script.sh]: maybe we need this line again or something like this to get back to the main extension directory?

**Comment by thiemowmde** [comment]
+1, but what's that failure?

**Comment by ricordisamoa** [comment]
> +1, but what's that failure?

What @filbertkm said.
Anyway PHP and NPM tests should really run as separate jobs

**Comment by manicki** [comment]
I guess this should be simply closed, as the code has been migrated to Gerrit, and there is eslint config in place out there?

**Pull request closed without merging**